kong/dao/schemas/apis.lua

```diff
@@ -163,7 +163,8 @@ return {
     request_path = {type = "string", unique = true, func = check_request_path},
     strip_request_path = {type = "boolean", default = false},
     upstream_url = {type = "url", required = true, func = validate_upstream_url_protocol},
-    preserve_host = {type = "boolean", default = false}
+    preserve_host = {type = "boolean", default = false},
+    retries = {type = "number", default = 5, func = check_retries},
```

Comment:
Maybe the previous `row.retries = 5` in the Cassandra migration was not set because the expectation was that re-updating the same entity would populate the default values.

Not sure if this really happens, and if it doesn't, should it be fixed? This needs to be tested.